docs/architecture/seo.md

```diff
@@ -1,46 +1,47 @@
+
+The SEO implementation in Spartacus includes the following:
+
 - Stateful URLs
 - Configurable URLs
-- Indexable Page response (SSR)
-- HTML tags
-  - title
-  - description
-  - robots
+- Indexable Pages
+- HTML Tags
+  - Page Meta Resolvers
+  - Title Resolver
+  - Description Resolver
+  - Robots Tag
 
 ## Stateful URLs
-Using URLs to have a stateful address for each piece of the storefront services a multiple goals. It will not only help users to better navigate the storefront, but also crawlers will be able to crawl each and every page. Crawlers can identify more pages that can shared through social media, bots, or search indexes. 
 
-In order to serve a stateful URL for everyting, we're covering a number of features:
-- Deeplinks can be used to address any page
-- The URL routing configuration can take the multi-site context into account so that a stateful URL for special variants of the storefront can be launched and cached. 
+Spartacus provides a URL with a stateful address for every piece of the storefront. This makes it easier for users to navigate the storefront, and also allows web crawlers to index each and every page. As a result, more pages can be shared through social media, bots, and search indexes.
 
-### Configurable URLs
-URLs for Content pages are configurable in the CMS by using a CMS page label. This is a page label that doesn't allow for translation. 
+To serve stateful URLs for everything, we allow deep links to address any page. The URL routing configuration can also take into account a multi-site context, so that stateful URLs for special variants of the storefront can be launched and cached.
 
-URLs for non-content pages are configurable in Spartacus. This is mainly related to product and category pages. Attributes, such the product name can be configured to be part of the URL. 
+## Configurable URLs
 
-i.e. the default configuration for a product page is storefront.com/product/1234, but customers can improve the URL in order to add the product or cateogry title to the URL.
+You can configure URLs for content pages by using a CMS page label in the CMS (back end). These page labels cannot be localized.
 
-Configurable URLs will help to improve SEO in general, but also help to migrate an existing solution to Spartacus; customers can keep their existing URLs and configure the equivalent URLs in spartacus. 
+You can configure URLs for non-content pages in Spartacus. These are mainly related to product and category pages. You can configure attributes, such as the product name, to be part of the URL. For example, the default configuration for a product page is `storefront.com/product/1234`, but you can modify the URL to include the product or category title. 
```

Comment:
"but you can modify the URL to include the product or category title" should be:

"but the URL can be configured to include product related data, such as the product or category title."

Reply:
Fixed.